### PR TITLE
add my Profile Card

### PR DESCRIPTION
**Pull request by pauloeduardomc**
Add my Profile Card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -109,7 +109,39 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ________ Contributor's card END ________  -->
-        <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
```
Comment (eligifty-coder) [inline index.html:112]: We do not want you to delete this line.

File: index.html [hunk 2]
```diff
@@ -109,7 +109,39 @@ <h2>A project for first-time contributions</h2>
...
+                      </li>
+                      <li>
+                        <a href="https://yukaichou.com/gamification-examples/octalysis-complete-gamification-framework/" target="_blank" title="Gamification & Behavioral Design">
+                          Gamification & Behavioral Design
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+          <!-- ________ Contributor's card END ________  -->
```
Comment (eligifty-coder) [inline index.html:144]: Kindly take your entire card and paste it below the line that says "Paste Your Card Directly Below this Line"

**Review by eligifty-coder — CHANGES_REQUESTED** [review]
Kindly effect the corrections requested

**Comment by Syknapse** [comment]
Manually fixed the problem and merged the card. Closing this PR now

**Pull request closed without merging**